js/index.js

```diff
@@ -71,7 +71,7 @@ function setDownloadURI() {
 function updateImage() {
   var query_string = generateQueryString();
   var preview = document.getElementById("preview");
-  preview.src = "../img/coderdojocertificate.svg" + query_string;
```

Comment:
Javascript is executed relative to the html file is is in. You can include many JS files from many places and they all use your html file location. @Alanh168